DataFormats/FEDRawData/interface/FEDNumbering.h

```diff
@@ -44,6 +44,8 @@ class FEDNumbering {
      MAXTotemTriggerFEDID = 577,
      MINTotemRPFEDID = 578,
      MAXTotemRPFEDID = 581,
+     MINCTPPSDiamondFEDID=582,
+     MAXCTPPSDiamondFEDID=583,
```

Comment:
Matched to 9_X_Y in 8057d679ab272677c17e41196b72fd85bcfd6aaf, but as discussed in https://github.com/cms-sw/cmssw/pull/17815#issuecomment-285869486 these FEDs were not part of the GRs in 2016.